app/src/main/java/org/fossasia/openevent/general/MainActivity.kt

```diff
@@ -121,18 +121,24 @@ class MainActivity : AppCompatActivity() {
     private fun loadFragment(fragment: Fragment) {
         supportFragmentManager.beginTransaction()
                 .replace(R.id.frameContainer, fragment, fragment::class.java.name)
-                .addToBackStack(null)
```

Comment:
Why is this removed?

Reply:
With this, App will not exit with `super.onBackPressed()`. Need to use `finish()` explicitly for exit the app.

Reply:
Without this, we don't have state saving of fragments